contents/docs/cdp/advanced.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Destinations: advanced usage"
+showTitle: true
+---
+
+## Customizing payload
+
+You can template destination output using curly braces `{}`.
+
+For example, given this PostHog event:
+
+```ts
+{
+    event: {
+        name: "$pageview",
+        properties: {
+            $current_url: "https://posthog.com"
+        }
+    },
+    person: {
+        name: "max@posthog.com",
+        properties: {
+            email: "max@posthog.com",
+            first_name: "Max",
+            last_name: "the Hedgehog"
+        }
+    }
+}
```

Comment:
This is actually my bad but the properties here are not correct
```suggestion
{
    event: {
        event: "$pageview",
        distinct_id: "1234",
        timestamp: "2024-01-01T12:00:00",
        properties: {
            $current_url: "https://posthog.com"
        }
    },
    person: {
        name: "max@posthog.com",
        properties: {
            email: "max@posthog.com",
            first_name: "Max",
            last_name: "the Hedgehog"
        }
    }
}
```

Reply:
thank you!